webapp/client/public/css/components.css

```
@@ -256,32 +256,44 @@
 }
 
 .details-card .card-header .details-icon {
-    width: 14px;
-    min-width: 14px;
+    width: 12px;
+    min-width: 12px;
     height: var(--lineheight-default);
     padding: 0;
-    margin-top: .375rem;
+    margin-top: 0;
     background:  url("/icons/details_arrow_down.svg") no-repeat left;
 }
 
-.details-card .card-header .collapsed:hover .details-icon{
+.details-card .card-header .collapsed:hover .control-show-more {
     background:  url("/icons/details_arrow_right_hover.svg") no-repeat left;
 }
 
-.details-card .card-header .collapsed button:focus .details-icon{
+.details-card .card-header .collapsed button:focus .control-show-more {
     background:  url("/icons/details_arrow_right_focus.svg") no-repeat left;
 }
 
-.details-card .card-header .collapsed .details-icon{
+.details-card .card-header .collapsed .control-show-more {
     background:  url("/icons/details_arrow_right.svg") no-repeat left;
 }
 
 .details-card .card-body {
     border: 0;
 }
 
-.details-card .details-content{
-    padding: 1em 1em 0 0;
+.details-card .details-content {
+    padding: var(--spacing-02) 1em var(--spacing-02) 0;
+}
+
+.details-card .details-content p:last-child {
+    margin-bottom: 0;
+}
+
+.details-card .details-content ul:last-child {
+    margin-bottom: 0;
+}
+
+.details-card .details-content ul li:last-child {
+    margin-bottom: 0 !important;
```

I'm not 100% sure, but couldn't this be a single rule?

```suggestion
.details-card .details-content :last-child {
    margin-bottom: 0;
}
```

This should cover all last-children inside the `.details-content`.

---

Thank you! I didn't know you could just write `:last-child` without another selector before it.